clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -336,9 +336,10 @@ def warn_anyx86_excessive_regsave : Warning<
   " with attribute 'no_caller_saved_registers'"
   " or be compiled with '-mgeneral-regs-only'">,
   InGroup<DiagGroup<"excessive-regsave">>;
-def warn_arm_interrupt_calling_convention : Warning<
-   "call to function without interrupt attribute could clobber interruptee's VFP registers">,
-   InGroup<Extra>;
+def warn_arm_interrupt_vfp_clobber : Warning<
+   "calling a VFP-enabled function from an interrupt could clobber the "
```

Comment:
Unless "VFP-enabled" is some Arm defined, specific spelling, I would write it as "VFP enabled" without the "-".

Reply:
It's a phrasal adjective which would normally be hyphenated, but I will consider how to rephrase so it's not needed.